schemas/event.yml

```diff
@@ -276,3 +277,18 @@
 
         This is mainly useful if you use more than one system that assigns
         risk scores, and you want to see a normalized value across all systems.
+
+    - name: ingested
+      level: core
```

Comment:
extended?

Reply:
I would like to introduce it directly as core. From the feedback we've seen, this timestamp is considered more useful than `event.created`.

A few reasons why that is so:

* It's another system's timestamp, which can help detect tampering of the clock on the monitored machine
* It can also be used to detect slowdowns in the overall pipeline, assuming no tampering

PRs such as https://github.com/elastic/beats/pull/14001 could also help populating it broadly and reliably, without having to revisit all modules or all beats.

If you have strong feelings and would really prefer to start by introducing as extended, I can go with that, in order to get this in quickly. But I think it would send the wrong message wrt to this timestamp's importance vs `event.created`.

Reply:
I'll wait for your response on this, and would like to merge this tomorrow if possible

Reply:
SGTM